subsystems/PistonControl.java

```diff
@@ -0,0 +1,22 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class PistonControl extends SubsystemBase{
+    private DoubleSolenoid[] doubleSolenoidPistons;
+
+    public PistonControl(int[] pistonIDS) {
+        for(int i = 0; i < pistonIDS.length; i+=2) {
+            doubleSolenoidPistons[i] = new DoubleSolenoid(i, i+1);
```

Comment:
This shouldn't be `i` and `i + 1`. It should be `pistonIDs[i]` and `pistonIDs[i + 1]`

Reply:
the `+=1` will actually cause `doubleSolenoidPistons[i]` to index too far if there is more than 1 piston